### PR TITLE
revert multithread georef

### LINKED ISSUE #41770: Georeferencer hangs on start georef (3.18 and master)
Georeferencer hangs after putting up Progress indicator window when told to start georeferencing.
QGIS unresponsive, need to kill process.

Happens in 3.18.0-1 release version (OSGeo4Win) and 3.19 master. Works OK (phew!) on 3.16.4 LTR.
New clean user profile.

Screenshot below.
Example tiff and .points file causing the problem available [in my DropBox](https://www.dropbox.com/s/pwc1l1yhn8fc3io/georeferencer_crash.zip?dl=0) (32 Mb).
I'm using thin plate spline, EPSG:2951 (but also tried 3857), output tiff compress=DEFLATE.

Have found issues posted here regarding *loading* a raster in georeferencer. This is different: not at load, this is after telling georef to go and warp the raster.

Screen_Capture_-_23_Feb__7_34_am

### PR DESCRIPTION
**Pull request by roya0045**
## Description

Fixes #41770

Should fix the issue until a better implementation of the feedback is found.

### CODE REVIEW

**Comment by roya0045** [comment]
Would need backport to 3.18 if sufficient to fix said issue.

File: src/app/georeferencer/qgsimagewarper.cpp [hunk 1]
```diff
@@ -240,7 +240,8 @@ int QgsImageWarper::warpFile( const QString &input,
   progressDialog->raise();
   progressDialog->activateWindow();
 
-  eErr = oOperation.ChunkAndWarpMulti( 0, 0, destPixels, destLines );
+  eErr = oOperation.ChunkAndWarpImage( 0, 0, destPixels, destLines );
+//  eErr = oOperation.ChunkAndWarpMulti(0, 0, destPixels, destLines);
```
Comment (m-kuhn) [inline src/app/georeferencer/qgsimagewarper.cpp:244]:
```suggestion
  eErr = oOperation.ChunkAndWarpImage( 0, 0, destPixels, destLines );
```

Comment (roya0045) [reply src/app/georeferencer/qgsimagewarper.cpp:244]: Strangely I cannot apply the suggestion in github. Though leaving the comment might not be this bad, I think we should maybe mention why it doesn't work instead, for an eventual followup. Does that sound like a better option?

Comment (m-kuhn) [reply src/app/georeferencer/qgsimagewarper.cpp:244]: The normal way in the QGIS codebase is to rip off unused code. I think it's better to stick to the standards and open an issue.

Comment (roya0045) [reply src/app/georeferencer/qgsimagewarper.cpp]: Very well, text removed. Thanks

**Comment by roya0045** [comment]
Thanks guys.

**Pull request merged**